Tests/tests_end_to_end/content/xsoar_saas/README.md

```diff
@@ -0,0 +1,35 @@
+# XSOAR-NG End to End Tests
+The purpose of XSOAR-NG end-to-end tests is to run basic regression tests for specific integrations. The tests run against a real tenant of XSOAR-NG.
```

Comment:
lets add this to the same doc as xsiam, not sure devs will know to look in both places

Reply:
already exist in xsiam README